v2/bigtable-cdc-to-hbase/src/main/java/com/google/cloud/teleport/v2/templates/BigtableToHbasePipeline.java

```diff
@@ -0,0 +1,250 @@
+import java.util.List;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.ExperimentalOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.client.RowMutations;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Bigtable change stream pipeline to replicate changes to Hbase. Pipeline reads from a Bigtable
+ * change stream, converts change stream mutations to their nearest Hbase counterparts, and writes
+ * the resulting Hbase row mutations to Hbase.
+ */
+@Template(
+    name = "bigtable-cdc-to-hbase",
+    category = TemplateCategory.STREAMING,
+    displayName = "Bigtable CDC to HBase Replicator",
+    description = "A streaming pipeline that replicates Bigtable change stream data to HBase",
+    optionsClass = BigtableToHbasePipeline.BigtableToHbasePipelineOptions.class,
+    flexContainerName = "bigtable-cdc-to-hbase",
+    contactInformation = "https://cloud.google.com/support")
+public class BigtableToHbasePipeline {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BigtableToHbasePipeline.class);
+  private static final String USE_RUNNER_V2_EXPERIMENT = "use_runner_v2";
+
+  /** Options to run pipeline with. */
+  public interface BigtableToHbasePipelineOptions
+      extends DataflowPipelineOptions, ExperimentalOptions {
+    /** Bigtable change stream configs. */
+    @TemplateParameter.Text(
+        description = "Bigtable project id",
+        helpText = "GCP project id that bigtable is under")
+    String getBigtableProjectId();
+
+    void setBigtableProjectId(String bigtableProjectId);
+
+    @TemplateParameter.Text(
+        description = "Bigtable instance",
+        helpText = "Bigtable instance to stream changes from")
+    String getInstanceId();
+
+    void setInstanceId(String instanceId);
+
+    @TemplateParameter.Text(
+        description = "Table id",
+        helpText = "The name of the Bigtable table whose changes to stream")
+    String getTableId();
+
+    void setTableId(String tableId);
+
+    @TemplateParameter.Text(
+        description = "App profile id",
+        helpText =
+            "Bigtable's app profile id needs to have single-cluster routing with single-row transactions allowed")
+    @Default.String("default")
+    String getAppProfileId();
+
+    void setAppProfileId(String appProfileId);
+
+    /** Hbase specific configs. Mirrors configurations on hbase-site.xml. */
+    @TemplateParameter.Text(
+        description = "Zookeeper quorum location",
+        helpText = "Zookeeper quorum location, corresponds to hbase.zookeeper.quorum")
+    String getHbaseZookeeperQuorum();
```

Comment:
please refer to other pipelines, we usually take host and port.

Reply:
Changed signature to host + port